src/tools/wasm-interp.cc

```diff
@@ -100,6 +101,9 @@ static void ParseOptions(int argc, char** argv) {
                    "Assume input module is WASI compliant (Export "
                    " WASI API the the module and invoke _start function)",
                    []() { s_wasi = true; });
+  parser.AddOption('e', "env", "ENV",
+                   "Pass the given environment string in the WASI runtime",
+                   [](const std::string& argument) { s_wasi_env = argument; });
```

Comment:
Probably want this to append to a vector, right? What is the expected format for the ENV string?